README.md

```diff
@@ -114,16 +114,29 @@ Visualizing a robot description:
 
 Available robot descriptions ([gallery](https://github.com/robot-descriptions/awesome-robot-descriptions#gallery)) are listed in the following categories:
 
-- [Arms](#arms)
-- [Bipeds](#bipeds)
-- [Dual arms](#dual-arms)
-- [Drones](#drones)
-- [Educational](#educational)
-- [End effectors](#end-effectors)
-- [Humanoids](#humanoids)
-- [Mobile manipulators](#mobile-manipulators)
-- [Quadrupeds](#quadrupeds)
-- [Wheeled](#wheeled)
+- [Robot descriptions in Python](#robot-descriptions-in-python)
```

Comment:
The TOC here was only meant to navigate locally between descriptions. Are you suggesting adding a global TOC to the readme?